subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -156,6 +273,22 @@ static u8_t ticker_by_slot_get(struct ticker_node *node, u8_t ticker_id_head,
 	return ticker_id_head;
 }
 
+#if defined(CONFIG_BT_LL_SW)
```

Comment:
It will be used in new arch, the controller is not using it yet in the new arch as the porting is pending.